PyBall/models/draft/person.py

```diff
@@ -0,0 +1,20 @@
+from PyBall.models.pitch_hand import PitchHand
+from PyBall.models.bat_side import BatSide
+from PyBall.models.primary_position import PrimaryPostion
+
+
+class Person:
+    def __init__(self, id=None, fullName=None, link=None, firstName=None, lastName=None,
+                 birthDate=None, birthCountry=None, primaryPosition=None, batSide=None,
+                 pitchHand=None, nameSlug=None):
+        self.id = id
+        self.fullName = fullName
+        self.link = link
+        self.firstName = firstName
+        self.lastName = lastName
+        self.birthDate = birthDate
+        self.birthCountry = birthCountry
+        self.primaryPosition = PrimaryPostion(**primaryPosition)
```

Comment:
We should consider how to deal with nested objects like these that will throw errors on bad data (like with the Dominican Summer League). However the `or {}` approach adds a lot of repetition and room for error if and when we forget about it. It's not something to tackle in this PR but just something to keep in mind.